tests/testthat/test-downloadModule.R

```diff
@@ -45,6 +45,7 @@ test_that("downloadModule downloads and unzips a parent module", {
   d_expected <- moduleMetadata("LCC2005", tmpdir)$childModules %>%
     c(m, "data", "testthat") %>% sort()
 
+#   expect_equal(length(f), 42)
   expect_equal(length(f), 44)
```

Comment:
No it can't be because I haven't changed the test and it passes on other machines including travis.